src/log.py

```diff
@@ -570,26 +570,108 @@ def format(self, record):
         return msg
 
     def identify_data_to_redact(self, record):
+        # This matches stuff like db_pwd=the_secret or admin_password=other_secret
+        # (the secret part being at least 3 chars to avoid catching some lines like just "db_pwd=")
+        # Some names like "key" or "manifest_key" are ignored, used in helpers like ynh_app_setting_set or ynh_read_manifest
+        secret_keys = 'pass|secret|token|salt'
+        secret_keys = f"{secret_keys}|{secret_keys.upper()}|pwd"
+        # Avoid to match some non relevant var compound with 'key'
+        secret_keys_regex = f"\w*({secret_keys})\w*|local key_?| key\w*|\w+key\w*|\w*KEY\w+|\w+KEY"
+        operator_regex = r'(\s+\-\-value)?='
```

Comment:
```suggestion
        operator_regex = r'(\s+\-\-value)?['"'"'"]?\s+(=|:|=>)\s+'
```
We probably should support other assignment operator, but it means better whitelist otherwhise evrything will be redacted ><